src/components/nft-viewer/nft-viewer.tsx

```diff
@@ -78,11 +60,11 @@ export const NftViewer = ({
   const [sortFieldName, setSortFieldName] =
     useState<keyof NftCollection>('created_at');
 
-  const sections = useMemo(
+  const sections: NftSection[] = useMemo(
     () =>
       data
-        ?.map?.(it => ({...it, data: [it]}) as SectionElement)
-        ?.sort?.(arraySortUtil(sortDirection, sortFieldName)),
+        .map(item => ({...item, data: [{data: item.data}]}) as NftSection)
```

Comment:
What is the reason of duplicating the `item.data` field? You can get it from the `item` object

Reply:
This is for section list. Section list data should have data array.